src/fusor/fusion_caller_models.py

```diff
@@ -247,3 +261,22 @@ class Genie(FusionCaller):
     reading_frame: str = Field(
         ..., description="The reading frame status of the fusion"
     )
+
+
+class CIVIC(FusionKnowledgebase):
+    """Define parameters for CIVIC model"""
+
+    type: Literal[KnowledgebaseList.CIVIC] = KnowledgebaseList.CIVIC
+    model_config = ConfigDict(arbitrary_types_allowed=True)
+    vicc_compliant_name: str = Field(
+        ..., description="The VICC compliant name for the fusion"
+    )
+    five_prime_end_exon_coordinates: ExonCoordinate | None = Field(
```

Comment:
This can be a separate issue, but we should consider shortening names. For example, this could be shortened to five_prime_end_exon_coords`